### PR TITLE
hair gradients

### PR DESCRIPTION
**Pull request by WindowsErrors**
## About The Pull Request

hair gradients

you use hair spray to get them

Screenshot (1487)

## Why It's Good For The Game

notice you don't have them yet

here you go

now even the most plain looking femoid can look like a real cut of fuckable meat with this

## Changelog
:cl:
add: hair dye
add: trait to make you spawn with hair dye
/:cl:

### CODE REVIEW

**Comment by retlaw34** [comment]
hell yeah

**Comment by triplezeta** [comment]
a surprise to be sure

**Comment by HighPrincessErinys** [comment]
what kind of statement is "real cut of fuckable meat"

**Comment by tiramisuapimancer** [comment]
make it a loadout item instead of a quirk

**Comment by WindowsErrors** [comment]
@HighPrincessErinys 

elevator door opens, Adam Smasher is on the other side
less than a second after the door opens, Adam Smasher greets you with
https://youtu.be/HrpptCZf9js
Adam Smasher refuses to elaborate further
Adam Smasher enters the elevator you just left
Adam Smasher leaves

File: whitesands/code/modules/client/loadout/loadout_general.dm [hunk 1]
```diff
@@ -150,6 +150,12 @@
 	var/obj/item/toy/plush/among/temp = new path()
 	description = "[capitalize(pick(temp.among_colors))] sus."
 	qdel(temp)
+
+/datum/gear/hairspray
```
Comment (WindowsErrors) [inline whitesands/code/modules/client/loadout/loadout_general.dm:154]: @tiramisuapimancer ok

Comment (WindowsErrors) [reply whitesands/code/modules/client/loadout/loadout_general.dm:154]: https://youtu.be/r0bOOu6pbkQ

**Comment by triplezeta** [comment]
this was tested ingame and it appears to work just fine, aside from needing to reconnect in order to see the gradient. LGTM?

**Comment by WindowsErrors** [comment]
what's LGTM

**Comment by triplezeta** [comment]
Lets Get This Merged

**Pull request merged**